core/src/main/scala/org/locationtech/rasterframes/tiles/ProjectedRasterTile.scala

```diff
@@ -58,5 +59,11 @@ object ProjectedRasterTile {
 
   def unapply(prt: ProjectedRasterTile): Option[(Tile, Extent, CRS)] = Some((prt.tile, prt.extent, prt.crs))
 
-  implicit lazy val projectedRasterTileEncoder: ExpressionEncoder[ProjectedRasterTile] = ExpressionEncoder()
+  implicit lazy val projectedRasterTileEncoder: ExpressionEncoder[ProjectedRasterTile] = {
+    // Makes sure UDTs are registered first
+    RasterSourceUDT
+    TileUDT
+    CrsUDT
```

Comment:
Can this be done in a less ad-hoc manner? At least have a single function across the source that implements this pattern?